### PR TITLE
Header styling responsive

### PR DESCRIPTION
**Pull request by scar055**
responsive pop over menu voledig gestyled

### CODE REVIEW

**Review by ryanvvalkenhoef — APPROVED** [review]

File: packages/next-templates/src/components/ExampleHeader/ExampleHeader.tsx [hunk 1]
```diff
@@ -20,11 +35,39 @@ export const ExampleHeader = ({ ...props }: ExampleHeaderProps) => (
...
-        <Button appearance="secondary-action-button">
+        <Button className="example--header-search-secondary-button" appearance="secondary-action-button">
+          <Search /> Zoeken
+        </Button>
+        <Button className="example--header-search-subtle-button" appearance="subtle-button">
           <Search /> Zoeken
         </Button>
       </div>
     </div>
+    <dialog id="popovertest" className="example--header-popover-menu" role="dialog" popover="auto">
```
Comment (MeesoPost) [inline packages/next-templates/src/components/ExampleHeader/ExampleHeader.tsx:46]: VERANDER!!!!!!!!

Comment (scar055) [reply packages/next-templates/src/components/ExampleHeader/ExampleHeader.tsx:46]: WHY ARE WE SCREAMING????!!!!

Comment (MeesoPost) [reply packages/next-templates/src/components/ExampleHeader/ExampleHeader.tsx:46]: ¯\_(ツ)_/¯

Comment (MeesoPost) [reply packages/next-templates/src/components/ExampleHeader/ExampleHeader.tsx]: ੧⟅ ︣ ʖ̫ ︣ ⟆୨

**Pull request merged**